### PR TITLE
AN7581: working I2C support

### PR DESCRIPTION
**Pull request by merbanan**

### CODE REVIEW

File: target/linux/airoha/dts/en7581.dtsi [hunk 1]
```diff
@@ -163,6 +163,48 @@
...
+
+			/* 400 kHz */
+			clock-frequency = <400000>;
+			resets = <&scuclk EN7581_I2C_MASTER_RST>;
+			reset-names = "i2c0";
+
+			#address-cells = <1>;
+			#size-cells = <0>;
+
+			status = "enable";
```
Comment (LorenzoBianconi) [inline target/linux/airoha/dts/en7581.dtsi]: @merbanan should it be disabled?

File: target/linux/airoha/files/arch/arm64/boot/dts/airoha/en7581.dtsi [hunk 2]
```diff
@@ -163,6 +163,48 @@
...
+
+			/* 400 kHz */
+			clock-frequency = <400000>;
+			resets = <&scuclk EN7581_I2C_MASTER_RST>;
+			reset-names = "i2c0";
+
+			#address-cells = <1>;
+			#size-cells = <0>;
+
+			status = "enable";
```
Comment (LorenzoBianconi) [inline target/linux/airoha/files/arch/arm64/boot/dts/airoha/en7581.dtsi]: @merbanan same here. I guess you can just enabled it in en7581-evb.dts

File: target/linux/airoha/dts/en7581.dtsi (hunk 1 above)
Comment (merbanan) [reply target/linux/airoha/dts/en7581.dtsi]: Yes, I forgot to fix the dts. For the ref board only i2c0 should be enabled. I'll address this.

Comment (LorenzoBianconi) [reply target/linux/airoha/dts/en7581.dtsi]: @merbanan ack, thx. Can you please rebase on top of my branch?

Comment (merbanan) [reply target/linux/airoha/dts/en7581.dtsi]: I think I have now.

**Comment by LorenzoBianconi** [comment]
@merbanan have you tested with any slave devices?

**Comment by LorenzoBianconi** [comment]
@merbanan I think you are missing your SoB

**Comment by merbanan** [comment]
> @merbanan have you tested with any slave devices?

On my device and I think evb there is a laser driver connected to the i2c0 bus. This device answers on the addresses it should answer. i2c1 has not been tested, its needs a pin mux to work.

**Comment by merbanan** [comment]
> @merbanan I think you are missing your SoB

Yes, I will address that.

**Pull request merged**